test/test-manager/src/tests/mod.rs

```diff
@@ -80,7 +80,7 @@ pub async fn cleanup_after_test(
     rpc_provider: &RpcClientProvider,
 ) -> anyhow::Result<()> {
     log::debug!("Resetting daemon settings after test");
-    // Check if daemon should be restarted
+    // Check if daemon should be restarted.
```

Comment:
I'd avoid touching this fn if I were you, it's going to conflict with my `test-manager-fixes` PR